backend/src/main/java/ai/verta/modeldb/versioning/blob/diffFactory/BlobDiffFactory.java

```diff
@@ -39,6 +42,18 @@ public BlobExpanded getBlobExpanded() {
       final Builder builder =
           BlobDiff.newBuilder().addAllLocation(Arrays.asList(location.split("#")));
       builder.setStatus(DiffStatus.MODIFIED);
+      //hack
```

Comment:
```suggestion
      //FIXME : remove hack
```

so that it pops up in IDE list